### DISCUSSION TITLE
Getting error while sign in

### DISCUSSION THREAD

**Discussion by pvgavfale** [Genuitec/Copilot4Eclipse-community#21] [Help] [answered]
![Screenshot 2024-09-20 161155](https://github.com/user-attachments/assets/eeef4134-6242-4041-be6c-4856e54bd732)
Getting this error while sign in

Spring Tool Suite 4 

Version: 4.25.0.RELEASE
Build Id: 202409101855

**Comment by wayneparrott** [comment]
Thanks for sharing!
I just setup STS 4.25.0/windows.
I installed the latest Copilot4Eclipse v1.2.2 (released today) and functions appear to be working properly. 
I would recommend you uninstall Copilot4Eclipse and then reinstall it. Something fundamentally looks goofed by that error msg your shared.

Let us know if a installing the new 1.2.2 update of Copilot4Eclipse resolves this issue.

Wayne

**Reply by pvgavfale** [reply]
Thank you for you reply, I tried the steps you mentioned and now i have v1.2.2


![co-pilot2](https://github.com/user-attachments/assets/cf7b0462-be26-4fe9-8096-00c349961521)

But when my sts got restarted after installation , i got this error


![co-pilot1](https://github.com/user-attachments/assets/49622090-1adf-419b-99cd-0061e97aa08b)

**Comment by brianvfernandes** [comment]
@pvgavfale could you please attach the entire error log from the workspace here? You can export it from the Error Log view in STS - this will give us more details on exactly what you're running into. Thanks!

**Reply by pvgavfale** [reply]
[copilot-exception.txt](https://github.com/user-attachments/files/17083632/copilot-exception.txt)
[copilot-exception-09-20.txt](https://github.com/user-attachments/files/17083633/copilot-exception-09-20.txt)

**Reply by brianvfernandes** [reply]
This helps - it looks like you have a different plugin / version configuration resulting in some missing classes. 

1. How did you install STS? I'm assuming this is a fresh install and not an update from a previous version. Did you install it from the STS4 all-in-one installer or add it to Eclipse from an update site?
2. Have you installed any plugins in addition to Copilot4Eclipse?
3. Can you please share the `bundles.info` file with us? You'll find it in the `[STS installation folder]/configuration/org.eclipse.equinox.simpleconfigurator` folder - this will let us know what plugins you have in your install and corresponding versions. 

Thanks!

**Reply by pvgavfale** [reply]
I was having STS which was previously installed. I just updated to latest version. I had copilot 1.2.1 now upgraded to 1.2.2
[bundles.info.txt](https://github.com/user-attachments/files/17083759/bundles.info.txt)

**Reply by brianvfernandes** [reply, marked as answer]
Thank you! We'll evaluate your bundles.info file and figure out what dependencies need to be corrected. This will take some time, however, so the most expedient way for you to fix this immediately would be to start with a fresh STS install, rather than using the version you have updated; given our testing with STS 4.25, the fresh install does have plugins that are compatible with Copilot4Eclipse. Thanks again for sharing these details with us.

**Reply by pvgavfale** [reply]
I did a fresh install of STS and with that  it is working now. I suspect with upgrade of STS probably cause some issue in Copilot4Eclipse. For now I am good to go.
Thank you @brianvfernandes @wayneparrott

**Comment by wayneparrott** [comment]
Checking in to see if you have worked around this issue.
We are working to setup an env to hopefully replicate the issue. As Brian shared the bundle info was useful. Thanks.

W
